### PR TITLE
Fix/missing env variables

### PR DESCRIPTION
**Pull request by chavda-bhavik**
**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Adds `SENTRY_DSN` to `docker-compose.yml` and `.env.example` file
- Adds `NEW_RELIC_APP_NAME` & `NEW_RELIC_LICENSE_KEY` to `docker-compose.yml` and `.env.example` file

**Why this change was needed?** (You can also link to an open issue here)
https://github.com/novuhq/novu/issues/1435

- **Other information**:

### CODE REVIEW

**Review by scopsy — COMMENTED** [review]
I think I would just solve it by having a default "empty" string value inside `apps/api/src/config/env-validator.ts` and remove the empty values added on .env.example and also the docker-compose. What do you think?

File: docker/.env.example [hunk 1]
```diff
@@ -35,3 +35,8 @@ WEB_CONTEXT_PATH=
 API_CONTEXT_PATH=
 WS_CONTEXT_PATH=
 WIDGET_CONTEXT_PATH=
+
+# Analytics
```
[the remark this answers is not in the document]
Comment (chavda-bhavik) [reply docker/.env.example:39]: @scopsy We can do `setting "empty" string value inside apps/api/src/config/env-validator.ts` as a precaution, 

But removing keys from `docker-compose.yml` file will lead to problems mentioned in *description* of https://github.com/novuhq/novu/issues/1435. 
And If we're taking environment variables than I believe it's good to add them in `.env.example`. 

At the end open to your approach.

[the remark this answers is not in the document]
Comment (scopsy) [reply docker/.env.example:39]: Let's go!

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**